column.go

```diff
@@ -76,6 +76,9 @@ func (c *Column) Path() []string { return c.path[1:] }
 // Name returns the column name.
 func (c *Column) Name() string { return c.schema.Name }
 
+// FieldID returns column field id
+func (c *Column) FieldID() int32 { return c.schema.FieldID }
```

Comment:
What do you think about using `int` for the public interface instead of `int32`. My reasoning is that fact that it's a 32-bit integer is more of an implementation detail, but it doesn't have to leak into the exported APIs.

Reply:
`int` sounds good to me.